members/forms.py

```diff
@@ -1,19 +1,36 @@
 from django import forms
+from django.utils import timezone
 from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
+from iquise.utils import random_password
 
-from members.models import Person
+def this_year():
+    return timezone.now().year
 
+class JoinForm(forms.Form):
+    LEVELS = ("Highschool", "Undergraduate", "Graduate", "PostDoc", "Professor", "Professional", "Retired")
+    first_name = forms.CharField(max_length=30)
+    last_name = forms.CharField(max_length=30)
+    email = forms.EmailField(help_text='Please use your university email if possible.')
```

Comment:
we should make this stronger. "You must use a university email." 

since we don't accept people who put a gmail...